src/java/org/apache/cassandra/repair/autorepair/AutoRepair.java

```diff
@@ -223,7 +224,11 @@ public void repair(AutoRepairConfig.RepairType repairType)
                 }
 
                 // Separate out the keyspaces and tables to repair based on their priority, with each repair plan representing a uniquely occuring priority.
-                List<PrioritizedRepairPlan> repairPlans = PrioritizedRepairPlan.build(keyspacesAndTablesToRepair, repairType, shuffleFunc);
+                List<PrioritizedRepairPlan> repairPlans = PrioritizedRepairPlan.build(keyspacesAndTablesToRepair, repairType, shuffleFunc, primaryRangeOnly);
+
+                int keyspaceRepairPlansSofar = 0;
```

Comment:
It is cumbersome to identify if a keyspace has been completely repaired or not, due to the priority-based allotment we have. In most cases, when priority is not set, it is 1:1. I know it is hard for end users to differentiate between keyspace vs. keyspaceRepairPlans, but we can at least provide them some metrics that exhibits expected and gradual increase